js/javascript.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function() {
+
+	// when user .click #submit-btn, perform function change
+	$("#submit-btn").click(change);
+
+	// store input in var city
+	var city;
+
+	function change(event){
+		// prevent form submission
+		event.preventDefault();
+		// call user input using .val() to get string value
+		// change all input to lower case
+		city = $("#city-type").val().toLowerCase();
+
+		// add if / else if / else conditionals
+		// use || operator to allow multiple string values
+		if (city === "nyc" || city === "new york" || city === "new york city"){
+			$("body").css("background-image" , "url(images/nyc.jpg)");
```

Comment:
Great job using jQuery's `css` method to change the background. Just so you're aware, these backgrounds were also available through different class names in `styles.css`, so you could have equally changed the background, say for NY, like this:

```
$('body').attr('class', 'nyc')
```

That would set the `class` property on the body to `nyc`, which is defined in the css file. Either way, just wanted you to be aware of your options! Great job here :)